release-notes/DxMs/DataAggregator_change_log.md

```diff
@@ -1,12 +1,18 @@
 ---
-uid: DataAggregator_1.0.0
+uid: DataAggregator_change_log
 ---
 
-# Data Aggregator 1.0.0
+# Data Aggregator change log
 
-## New features
+#### 13 February 2023 - Enhancement - DataAggregator 3.0.2 - Debug UI shows job names instead of job IDs [ID_38697]
 
-#### New Data Aggregator module [ID_34725] [ID_34825] [ID_34914] [ID_34986] [ID_35047] [ID_35072] [ID_35450] [ID_35495]
+In the debug UI of Data Aggregator (accessible via `http(s)://<DMA hostname or IP>.<Data Aggregator port>/debug>`) the different tab pages will now show the job name instead of the job ID, so that it is easier to find a specific job.
```

Comment:
`http://<Data Aggregator hostname or IP>:<Data Aggregator port>/debug/`

Data Aggregator does not support running on https directly.
Data Aggregator can run on another machine, does not have to be a DMA.
Should be a colon between hostname and port.
The url ends with a `/`.